app/src/lumen/java/com/cliqz/browser/subscription/TrialPeriodHandler.kt

```diff
@@ -0,0 +1,56 @@
+package com.cliqz.browser.subscription
+
+import android.os.AsyncTask
+import com.cliqz.browser.BuildConfig
+import com.cliqz.browser.utils.HttpHandler
+import com.revenuecat.purchases.Purchases
+import org.json.JSONObject
+import java.net.HttpURLConnection
+import java.net.URL
+
+private const val CONTENT_TYPE_JSON = "application/json"
+private val LUMEN_CREDENTIAL_URL = URL("https://auth-staging.lumenbrowser.com/get_credentials")
+
+private val HEADERS = mapOf("x-api-key" to BuildConfig.VPN_API_KEY)
+
+private val REQUEST_BODY = JSONObject()
+        .put("device_id", Purchases.sharedInstance.appUserID)
+        .put("revenue_cat_token", BuildConfig.REVENUECAT_API_KEY)
```

Comment:
This should be the per user revenue cat token returned by revenue cat, not the api-key.

Reply:
done